pipelines/LSSTCam-imSim/ApVerify.yaml

```
@@ -15,6 +15,12 @@ tasks:
     class: lsst.ap.association.DiaPipelineTask
     config:
       doPackageAlerts: True
+  rbClassify:
+    class: lsst.meas.transiNet.TransiNetTask
+    config:
+      modelFile: testModel
+      weightsFile: '/home/nima/meas_transiNet/tests/data/checkpoint.pth.zip'
+      connections.coaddName: 'goodSeeing'
```

Hard-coding the coadd name causes a lot of problems when pipelines must be customized for specific datasets:
```suggestion
      connections.coaddName: parameters.coaddName
```

---

I'm flattered, but it's not standard practice in DM to cite people for pointing out corrections on the PR. Just [write the commit as if you'd done it that way the first time](https://developer.lsst.io/work/flow.html#rebase-commits-from-code-reviews-rather-than-having-review-feedback-commits).